device-types/SNR/S2995G-12FX-UPS.yaml

```diff
@@ -0,0 +1,112 @@
+---
+manufacturer: SNR
+model: S2995G-12FX-UPS
+slug: snr-s2995g-12fx-ups
+part_number: SNR-S2995G-12FX-ups
+u_height: 1.0
+is_full_depth: false
+weight: 2.86
+weight_unit: kg
+airflow: right-to-left
+comments: '[SNR-S2995G-12FX](https://snr.systems/ru/catalog/setevoe-oborudovanie/kommutatory-agregacii-i-yadra/snr-s2995g/snr-s2995g-12fx)'
+console-ports:
+  - name: con0
+    label: CONSOLE
+    type: rj-45
+power-ports:
+  - name: power AC
+    type: iec-60320-c14
+    maximum_draw: 60
+    label: 100-240V; 50/60Hz; 1.0A
+  - name: battery DC
+    type: other
+    maximum_draw: 60
+    label: DC 11-13V, 4.5A
+power-outlets:
+  - name: usb
```

Comment:
USB-a ports as power-outlets is not something currently being modeled in this repo